biothings/hub/__init__.py

```diff
@@ -1546,7 +1546,7 @@ def quick_index(
         build_name = f"{datasource_name}_{random_string}"
         # # generate index_name if needed
         if not index_name:
-            index_name = f"_QUICK_{build_name}"
+            index_name = f"quick__{build_name}"
```

Comment:
@sengineer0 do we need to add this `quick__` prefix?

Reply:
not really, just added to help us determine which index created by quick index or not.
You think it is not necessary, right?

Reply:
I see. I would just remove it. `f"{datasource_name}_{random_string}"` should be enough, otherwise, it will be `f"{build_name}_timestamp_{random_string}"`. They should be easy to tell the difference. If later we notice any confusion, we can adjust.

Reply:
ok. I will update it soon. and the random_string already contains timestamp.

Reply:
It's fixed